### DISCUSSION TITLE
Latex template location

### DISCUSSION THREAD

**Discussion by mauriziopassariello** [quarto-dev/quarto-cli#714] [Q&A]
Hi,

I have a personalised latex template that I would like to use with Quarto to render pdf documents. Currently, I have this in the same wd as the qmd file I am working on. Is there a possibility to save it (where??) on my mac so that I can open a new file from that template (as you can do with rmd files)?

Thanks!

**Comment by cderv** [comment]
@mauriziopassariello I am wondering about your current usage: Is this a full latex template to replace Pandoc default one ? or is it composed of several resource files ? How are you using it in your YAML header today ? 

We'll have a feature soon for helping create and use custom format. Our current idea is to have a way to easily import shared formats. 
@dragonstyle do you think we'll have a possibility for importing from a central local storage ? Or maybe a place like Pandoc's user data dir to allow shared resources ? I know we talked about having them locally, but I can see how a user could benefit a shared place for its workflow.

**Reply by mauriziopassariello** [reply]
@cderv Thank you for your reply. Yes, it is a latex replacement template for the default pandoc. At the moment I have it saved on my computer. In the YAML I have this
format:
  pdf:
    template: <path to the .tex file>
This seems to work. However, I was wondering if I could save it somewhere specific so that when I open a new qmd file I can create a file from template in the beginning, as you can do for an rmd file.

I see probably you are working on something...

**Reply by dragonstyle** [reply]
In the initial version, we will likely just support searching in the current directory (e.g. we'll look next to the document for a folder that provides the configuration metadata and supporting files).

I think we do intend to ultimately support placing a format extension in the project root (perhaps in an `_extensions` folder) and we would resolve the format against that. This would make it very easy to define a format that is supported across an entire project and simple reference it like you would a built in format. Does that make sense?
